lib/cn/omf.h

```diff
@@ -275,21 +275,37 @@ OMF_SETGET(struct bloom_hdr_omf, bh_n_hashes, 8)
 
 /*****************************************************************
  *
- * Vblock header OMF
+ * Vblock footer OMF
  *
  ****************************************************************/
 
-#define VBLOCK_HDR_MAGIC ((u32)0xea73feed)
+#define VBLOCK_FOOTER_MAGIC ((u32)0xea73feed)
+#define VBLOCK_FOOTER_LEN   4096
```

Comment:
Can you make this compile-time computed similar to the hblock? You should then be able to remove the static_assert() below I think.

Reply:
I would leave it the way it is. The assert ensures that when one extends struct vblock_footer_omf {} it all stays within a page.